spec/support/error_page.rb

```diff
@@ -1,4 +1,4 @@
 class ErrorPage < SitePrism::Page
   set_url '/error'
-  set_url_matcher /\/error/
+  set_url_matcher(/\/error/)
```

Comment:
I've noticed in places that people have used `%r{/error/}` which seems like a nicer syntax for this case with paths. No need to change it, just something I saw & liked.